packages/access-api/src/types/provisions.ts

```diff
@@ -0,0 +1,30 @@
+import * as Ucanto from '@ucanto/interface'
+
+export type AlphaStorageProvider = 'did:web:web3.storage:providers:w3up-alpha'
+
+/**
+ * action which results in provisionment of a space consuming a storage provider
+ */
+export interface StorageProvisionCreation {
```

Comment:
Nit: I would prefer if we did not had `Storage` prefix, protocol is meant to allow you to add capability providers which can be for storage or other things sometimes for multiple things. It is true that `did:web:web3.storage:providers:w3up-alpha` is mostly for storage, yet it's not just storage it's also `upload/*` and `access/*`.

Reply:
At this layer of the types I wanted to model this particular use case really explicitly (e.g. with `space` and `account` vs `consumer` and `issuer`). In practice those get translated into a sql table that I think can accomodate the more general thing you're talking about over time. We could have many of these narrower types that mode [domain event](https://www.martinfowler.com/eaaDev/DomainEvent.html) like this that write into the same underlying thing.

> protocol is meant to allow you to add capability providers which can be for storage or other things sometimes for multiple things.
Because the underlying table accomodates this, I don't want to change this particular action type now. I'm not against revising it or adding other events in subsequent PR. or e.g. renaming the model to not have storage prefix but still maybe having this particular event be specific to adding a storage provider.

Reply:
I removed 'Storage' prefix from the model object itself https://github.com/web3-storage/w3protocol/pull/462/commits/e62faaaa3c69c75e8f81a7ec35e9abccb4192da6